packages/terra-list/tests/jest/__snapshots__/List.test.jsx.snap

```diff
@@ -117,34 +117,19 @@ exports[`should render with ariaDescribedBy 1`] = `
   <InjectIntl(List)
     ariaDescribedBy="list-help"
   >
-    <ListItem
-      hasChevron={false}
-      isSelectable={false}
-      isSelected={false}
+    <InjectIntl(ListItem)
```

Comment:
Missing dive

Reply:
We cant add dive() to this test case. as it tells can not be called on Host Components. Thanks